OC4IDS_Quality_Criteria_Checks_and_Metrics.ipynb

```diff
@@ -0,0 +1,1792 @@
+{
```

Comment:
<p>I don't understand why 2 collections need to be chosen and what the difference should be between them, some explanation is needed, or if there is explanation elsewhere it needs to be signposted here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-contracting/notebooks-oc4ids/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Explanation added